docs/user-guide/cai.rst

```diff
@@ -14,7 +16,7 @@ CAI allows training a harmless, but non-evasive AI assistant that engages with h
 .. _Constitutional AI (CAI): https://arxiv.org/abs/2212.08073
 
 CAI
-###############
+###
 The basic steps of CAI are described in this section and illustrated in the figure below (`Figure 1 <https://arxiv.org/abs/2212.08073>`_).
```

Comment:
Check link <https://arxiv.org/abs/2212.08073>. This link points to an arkiv article, Constitutional AI: Harmlessness from AI Feedback, not the "figure below" in this file.